### ISSUE TITLE
Scheduled posts don't post

### ISSUE THREAD

**Issue by BrettMN** [TryGhost/Ghost#8485]
Welcome to Ghost's GitHub repo! 👋🎉

Do you need help or have a question? Please come chat in Slack: https://ghost.org/slack 👫. Got an idea for a new feature? Please add it to our wishlist: http://ideas.ghost.org 🌟. Found a bug? Please fill out the sections below... thank you 👍

### Issue Summary

A summary of the issue and the browser/OS environment in which it occurs. 

### Steps to Reproduce

1. schedule a post in the future
2. wait till future post time passes
3. look at blog, see post is not there
4. in admin panel see post is still scheduled but time has past
5. edit post
6. change nothing and "Update Post"
7 reload blog, see post is listed
8 in admin panel see post was "published" at time set in step 1

here is an example:

![scheduled post issue](https://cloud.githubusercontent.com/assets/3488212/26414248/2febec00-4074-11e7-9bd1-4bd82f5fc457.gif)

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

### Technical details:

using the docker image with tag 0.11.9 on Ubuntu 16.04

saw it happening with the docker:0.11.8 as well

* Ghost Version: 0.11.9
* Node Version:
* Browser/OS: Chrome Desktop on Windows, Brave Mobile on ios, Safari Mobile on ios
* Database:  sql lite

**Comment by kirrg001** [comment]
Hey 👋

Sorry to hear you are having trouble with scheduling.
I have a couple of questions to be able to help you.

Does it never publish for you? or only sometimes?
Do you have a special cache in front of your blog?
Could you schedule a test post +3mins and provide me your Ghost server logs? Helpful would be shortly before the post should be published and a little after.

Thanks 🙃

**Comment by BrettMN** [comment]
Sorry I'm not sure how to get the server logs I did find a second way to reproduce it.

1. download 0.11.9 from https://github.com/TryGhost/Ghost/releases/tag/0.11.9
2. unzip
3. run `npm install --production` on the command line
4. run `npm start --production` on the command line
5. open a browser to http://localhost:2368/ghost
6. create an admin account
7. create and schedule post in the future, I used 5 minutes
9 return to "content" list and verify post is "scheduled"
10. wait till after post should be posted

11. refresh content window post is still "scheduled"
12. open second window to http://localhost:2368/ and see post only the "Welcome to Ghost" post

13. back in admin panel create a new post and publish 

14. in non admin tab refresh and see new post but scheduled post is still not published. 

here's a gif :) 


![scheduled post issue 2](https://cloud.githubusercontent.com/assets/3488212/26436625/70f5cee6-40dd-11e7-93ad-ba02f950b0d6.gif)


This was running Ghost on Windows 10 with Power Shell

**Comment by kirrg001** [comment]
Sorry, but I can't reproduce what you are experiencing, not on Windows 10, nor on my mac.

Could you please swing by our [slack channel](https://ghost.org/slack/)?
Then i can help you finding your server logs - without them i am not able to help you. 

If this turns out to be an actual issue with Ghost this issue can be reopened.

**Comment by BrettMN** [comment]
@kirrg001 is there an page in the documentation that says where the logs are?  so we don't have to be online at the same time?

**Comment by kirrg001** [comment]
If you start Ghost with e.g. `npm start --production`, you should see the server log in the shell if you request `localhost:2368/ghost`

**Comment by BrettMN** [comment]
Here is the log from starting ghost, 
deleted sample posts, 
reloading page, 
creating future post,
 waiting till time passed,
 reloading non admin window,
 creating new post and publishing immediately,
 returning to 'content' tab,
 reloading non admin tab seeing new post,
 editing and updating scheduled post, 
returning to 'content' tab seeing it as no longer scheduled and listed as published before the the post that was immediately published, 
refreshing non admin tab to see the scheduled post. 




```
Copyright (C) 2016 Microsoft Corporation. All rights reserved.

PS D:\Workspace\WipDeveloper\Ghost-0.11.9> npm start --production

> ghost@0.11.9 start D:\Workspace\WipDeveloper\Ghost-0.11.9
> node index

WARNING: Ghost is attempting to use a direct method to send email.
It is recommended that you explicitly configure an email service.
Help and documentation can be found at http://support.ghost.org/mail.

Ghost is running in production...
Your blog is now available on http://my-ghost-blog.com
Ctrl+C to shut down
127.0.0.1 - - [25/May/2017:15:41:20 +0000] "GET /ghost/ HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)
Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:20 +0000] "GET /ghost/vendor.min.css?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT 10.
0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:20 +0000] "GET /ghost/ghost.min.css?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT 10.0
; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:20 +0000] "GET /ghost/ghost.min.js?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT 10.0;
 Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:20 +0000] "GET /ghost/vendor.min.js?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT 10.0
; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"

ERROR: Unknown Request error.
 http://my-ghost-blog.com/ghost/img/ghosticon.jpg

 Error: Unknown Request error.
    at IncomingMessage.<anonymous> (D:\Workspace\WipDeveloper\Ghost-0.11.9\core\server\utils\image-size-from-url.js:79:31)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)

127.0.0.1 - - [25/May/2017:15:41:21 +0000] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:21 +0000] "GET /assets/css/screen.css?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Wi
n64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /assets/js/jquery.fitvids.js?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10
.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /assets/js/index.js?v=386a1e8313 HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64
; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/api/v0.1/settings/?type=blog%2Ctheme%2Cprivate HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/
5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "POST /ghost/api/v0.1/authentication/token HTTP/1.1" 200 - "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT
10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/api/v0.1/users/me/?include=roles&status=all HTTP/1.1" 200 723 "http://localhost:2368/ghost/" "Mozilla/5
.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/api/v0.1/notifications/ HTTP/1.1" 200 20 "http://localhost:2368/ghost/" "Mozilla/5.0 (Windows NT 10.0;
Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/api/v0.1/settings/?type=blog%2Ctheme%2Cprivate HTTP/1.1" 200 - "http://localhost:2368/ghost/2/" "Mozill
a/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:22 +0000] "GET /ghost/fonts/ghosticons.woff HTTP/1.1" 200 22496 "http://localhost:2368/ghost/ghost.min.css?v=386a1e8313" "Moz
illa/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:54 +0000] "GET /ghost/api/v0.1/posts/2/?status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localhost:2368/ghost/
2/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:55 +0000] "GET /ghost/api/v0.1/tags/?limit=all HTTP/1.1" 200 426 "http://localhost:2368/ghost/editor/2/" "Mozilla/5.0 (Window
s NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:55 +0000] "GET /ghost/api/v0.1/users/?limit=all&include=roles HTTP/1.1" 200 814 "http://localhost:2368/ghost/editor/2/" "Mozi
lla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:57 +0000] "PUT /ghost/api/v0.1/posts/2/?include=tags HTTP/1.1" 200 - "http://localhost:2368/ghost/editor/2/" "Mozilla/5.0 (Wi
ndows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:58 +0000] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:41:58 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:03 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/2/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:06 +0000] "GET /ghost/api/v0.1/posts/3/?status=all&staticPages=all&include=tags HTTP/1.1" 200 806 "http://localhost:2368/ghos
t/3/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:09 +0000] "DELETE /ghost/api/v0.1/posts/3/ HTTP/1.1" 204 - "http://localhost:2368/ghost/editor/3/" "Mozilla/5.0 (Windows NT 1
0.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:09 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/3/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:12 +0000] "GET /ghost/api/v0.1/posts/2/?status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localhost:2368/ghost/
2/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:15 +0000] "DELETE /ghost/api/v0.1/posts/2/ HTTP/1.1" 204 - "http://localhost:2368/ghost/editor/2/" "Mozilla/5.0 (Windows NT 1
0.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:15:42:16 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/2/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:08 +0000] "GET /ghost/api/v0.1/slugs/post/future%20post%20/ HTTP/1.1" 200 34 "http://localhost:2368/ghost/editor/" "Mozilla/5
.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:10 +0000] "POST /ghost/api/v0.1/posts/?include=tags HTTP/1.1" 201 497 "http://localhost:2368/ghost/editor/" "Mozilla/5.0 (Win
dows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:10 +0000] "GET /ghost/api/v0.1/slugs/post/future%20post/ HTTP/1.1" 200 36 "http://localhost:2368/ghost/editor/" "Mozilla/5.0
(Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:13 +0000] "PUT /ghost/api/v0.1/posts/4/?include=tags HTTP/1.1" 200 501 "http://localhost:2368/ghost/editor/4/" "Mozilla/5.0 (
Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:16 +0000] "PUT /ghost/api/v0.1/posts/4/?include=tags HTTP/1.1" 200 523 "http://localhost:2368/ghost/editor/4/" "Mozilla/5.0 (
Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:22 +0000] "PUT /ghost/api/v0.1/posts/4/?include=tags HTTP/1.1" 200 527 "http://localhost:2368/ghost/editor/4/" "Mozilla/5.0 (
Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:20:49 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/4/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:25:41 +0000] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:25:42 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:25:56 +0000] "GET / HTTP/1.1" 304 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:25:57 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:40 +0000] "GET / HTTP/1.1" 304 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:41 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:44 +0000] "GET /ghost/1/ HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko
) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/ghost/1/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) Apple
WebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "POST /ghost/api/v0.1/authentication/token HTTP/1.1" 200 - "http://localhost:2368/ghost/1/" "Mozilla/5.0 (Windows N
T 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "GET /ghost/api/v0.1/settings/?type=blog%2Ctheme%2Cprivate HTTP/1.1" 200 - "http://localhost:2368/ghost/1/" "Mozill
a/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "GET /ghost/api/v0.1/users/me/?include=roles&status=all HTTP/1.1" 200 723 "http://localhost:2368/ghost/1/" "Mozilla
/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "GET /ghost/api/v0.1/notifications/ HTTP/1.1" 200 20 "http://localhost:2368/ghost/1/" "Mozilla/5.0 (Windows NT 10.0
; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:45 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/1/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:46 +0000] "GET /ghost/api/v0.1/settings/?type=blog%2Ctheme%2Cprivate HTTP/1.1" 200 - "http://localhost:2368/ghost/1/" "Mozill
a/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:49 +0000] "GET /ghost/api/v0.1/tags/?limit=all HTTP/1.1" 200 426 "http://localhost:2368/ghost/editor/" "Mozilla/5.0 (Windows
NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:49 +0000] "GET /ghost/api/v0.1/users/?limit=all&include=roles HTTP/1.1" 200 814 "http://localhost:2368/ghost/editor/" "Mozill
a/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:54 +0000] "GET /ghost/api/v0.1/slugs/post/now%20post/ HTTP/1.1" 200 31 "http://localhost:2368/ghost/editor/" "Mozilla/5.0 (Wi
ndows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:57 +0000] "POST /ghost/api/v0.1/posts/?include=tags HTTP/1.1" 201 511 "http://localhost:2368/ghost/editor/" "Mozilla/5.0 (Win
dows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:27:59 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/5/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:02 +0000] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:02 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:05 +0000] "GET /ghost/api/v0.1/posts/4/?status=all&staticPages=all&include=tags HTTP/1.1" 200 527 "http://localhost:2368/ghos
t/4/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:07 +0000] "PUT /ghost/api/v0.1/posts/4/?include=tags HTTP/1.1" 200 524 "http://localhost:2368/ghost/editor/4/" "Mozilla/5.0 (
Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:08 +0000] "GET / HTTP/1.1" 200 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome
/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:09 +0000] "GET /favicon.ico HTTP/1.1" 200 - "http://localhost:2368/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/5
37.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
127.0.0.1 - - [25/May/2017:16:28:11 +0000] "GET /ghost/api/v0.1/posts/?page=1&limit=15&status=all&staticPages=all&include=tags HTTP/1.1" 200 - "http://localho
st:2368/ghost/editor/4/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36"
```

**Comment by kirrg001** [comment]
What is your configured blog timezone? And your server timezone?
Plus: could you please provide more details on how you are scheduling a post? Including your publish time.

**Comment by BrettMN** [comment]
the blog is configured with all defaults that came with 0.11.9 the server time zone is central daylight savings

I schedule post by following this https://help.ghost.org/hc/en-us/articles/227328167-Post-Scheduling

for the log and the last gif they are scheduled +3 minutes in the future as suggested here https://github.com/TryGhost/Ghost/issues/8485#issuecomment-303797404

**Comment by kirrg001** [comment]
I'm sorry, but i am still not able to reproduce this. Could you swing by our slack channel please?

**Comment by r3wt** [comment]
I have the same problem on a new installation of ghost running ubuntu 16.04 our content editor reports that scheduled posts are not going live on the website

**Comment by r3wt** [comment]
@gargol here is a reply to your request on the other thread for me to answer these questions

> What is your configured blog timezone? And your server timezone?
> Plus: could you please provide more details on how you are scheduling a post? Including your publish time.

configured server timezone on my Ubuntu 18.04 server is UTC
configured blog timezone says (GMT) UTC 

![image](https://user-images.githubusercontent.com/4483325/56429554-f363f780-6288-11e9-961f-016fbdec03bc.png)

![image](https://user-images.githubusercontent.com/4483325/56429504-c0216880-6288-11e9-8173-606a907fed2a.png)

> Does it never publish for you? or only sometimes?

No

> Do you have a special cache in front of your blog?

No 

> Could you schedule a test post +3mins and provide me your Ghost server logs? Helpful would be shortly before the post get's published and a little after.

Yes i can, but i am not sure how to retrieve the logs. can you advise me?

**Comment by r3wt** [comment]
@gargol @kirrg001 Can you shed some light on the process by which ghost publishes scheduled posts? does it rely on incoming requests to trigger a check, or does some backend process run? 

I have just tested using the instructions above, and the post published. however my content editor is still reporting scheduled posts never publishing, and indeed, all of his scheduled posts are just sitting unpublished. He is located in UK timezone and i am located in American timezone. could it be a timezone issue, even though we are using UTC?

**Comment by r3wt** [comment]
@gargol @kirrg001 They appear to be working now, and we have no idea why. I had content editor test again and the scheduled post published. If i ever figure out an isolated repro i'll share it with you.

**Comment by gargol** [comment]
Hi @r3wt ! Thanks for the info. 

Had a look at this as well, but wasn't able to reproduce the issue locally. Tried immediate scheduling (less than 10 minutes in the future), delayed one 15 minutes in the future and also a Ghost process restart in before scheduled publishing time.  

The scheduler is being run on the [server side](https://github.com/TryGhost/Ghost/blob/master/core/server/adapters/scheduling/SchedulingDefault.js) and it's queue is populated based on `published_at` data on the `post` resource.  

One additional thing that might help debugging this issue is setting the log level to `debug` in the Ghost configuration (needs restart):
```
    "logging": {
        "level": "debug"
    }
```
This might allow us to catch some more specific information if for example the job even pinged the URL to publish the post - https://github.com/TryGhost/Ghost/blob/42c472f/core/server/adapters/scheduling/SchedulingDefault.js#L202

**Comment by gargol** [comment]
@r3wt there was a [bug fix](https://github.com/TryGhost/Ghost/commit/2447335ab16e4e2595a2007fd1aa30217a23fbef) that landed in master today that might be related to issues with the scheduler you experienced. Will be released with Ghost version 2.22.0 tomorrow
